src/models/pledgeAdmins.model.js

```diff
@@ -27,7 +27,8 @@ function PledgeAdmin(app) {
       timestamps: true,
     },
   );
-
+  pledgeAdmin.index({ updatedAt: 1 });
+  pledgeAdmin.index({ createdAt: 1 });
```

Comment:
@mohammadranjbarz Where are these indexes useful?

Reply:
I thought indexing the timestamp will be useful for other queries indirect, but I wasn't sure,
If you think there is no benefit in indexing them  I remove from here and other schemas

Reply:
I removed unnecessary indexes